### PR TITLE
Update DOE, integrate feedback

### PR DESCRIPTION
**Pull request by knrds**

### CODE REVIEW

File: R/doeScreening.R [hunk 1]
```diff
@@ -52,11 +52,22 @@ doeScreening <- function(jaspResults, dataset, options, ...){
...
                        resolution = ifelse(options[["screeningType"]] == "PBdes",
                                            "III",
                                            "IV"),
-                       centers = options[["screeningCenterPoints"]])
+                       centers = ifelse(options[["screeningType"]] == "PBdes",
+                                        options[["screeningCenterPoints"]],
+                                        ifelse(options[["numberOfFactorsScreen2"]] == 0,
+                                               options[["screeningCenterPoints"]],
+                                               0)
+                                        )
```
Comment (vandenman) [inline R/doeScreening.R:69]: please don't use `ifelse` where you mean `if (condition) ... else ...`
```suggestion
                       centers = if(options[["screeningType"]] == "PBdes") { options[["screeningCenterPoints"]]
                                 } else if (options[["numberOfFactorsScreen2"]] == 0) { options[["screeningCenterPoints"]]
                                 } else { 0 }

```

File: R/doeScreening.R [hunk 2]
```diff
@@ -137,7 +148,9 @@ doeScreening <- function(jaspResults, dataset, options, ...){
     } else {
       daewr::DefScreen(m = options[["numberOfFactorsScreen3"]],
                        c = options[["numberOfFactorsScreen2"]],
-                       center = options[["screeningCenterPoints"]])
+                       center = ifelse(options[["numberOfFactorsScreen2"]] == 0,
+                                       options[["screeningCenterPoints"]],
+                                       0))
```
Comment (vandenman) [inline R/doeScreening.R:153]: same comment about `ifelse`

**Review by vandenman — CHANGES_REQUESTED** [review]

**Comment by vandenman** [comment]
@knrds please request another review whenever you're ready!

**Comment by knrds** [comment]
till when do I have time? I replaced all the ifelse's with if's in doeScreening.R, but I could check all my .R files for ifelse's till 15:00

**Comment by vandenman** [comment]
Ideally a little sooner than 15:00 though

**Comment by knrds** [comment]
that should be all ```ifelse```'s replaced!

**Review by vandenman — APPROVED** [review]

**Pull request merged**